src/System.CommandLine.DragonFruit/CommandLine.cs

```diff
@@ -21,19 +21,22 @@ public static class CommandLine
         /// </summary>
         /// <param name="entryAssembly">The entry assembly</param>
         /// <param name="args">The string arguments.</param>
+        /// <param name="entryPoint">Explicitly defined entry point</param>
         /// <returns>The exit code.</returns>
         public static async Task<int> ExecuteAssemblyAsync(
             Assembly entryAssembly,
-            string[] args)
+            string[] args,
+            string entryPoint)
```

Comment:
I would propose changing this parameter name to match the name of the MSBuild property.